Smith_Ben/Assignments/Basic Python Fund/drawstars.py

```diff
@@ -0,0 +1,18 @@
+# PART 1
+
+def draw_stars(x):
+	for count in range(0, len(x)):
+		print '*' * x[count]
```

Comment:
in this case, we (you) don't really care about the index. all you're really interested in is the _value_. So it's more efficient and simpler to iterate over the elements directly, rather than their indices, e.g.: `for ele in x: print '*' * x`